site/docs/layouts/shortcodes/sectionContent.html

```diff
@@ -0,0 +1,48 @@
+<div class="sectioncontent">
+    {{- $contentType := "all"}}
+    {{- if eq (.Get "type") "section" }}
+        {{- $contentType = "section" }}
+    {{- end}}
+    {{- $depthLimit := 2 }}
+    {{- with .Get "depth" }}
+        {{- $depthLimit = . }}
+    {{- end }}
+
+    {{- $startSection := .Page.Page }}
+    <ul>
+    {{- range .Page.Pages }}
+        {{- template "subsection-loop" dict "startSection" $startSection "currentSection" . "contentType" $contentType "depthLimit" $depthLimit "depthCnt" 0 }}
+    {{- end }}
+    </ul>
+</div>
+
+{{- define "subsection-loop" }}
+    {{- $startSection := .startSection }}
+    {{- $currentSection := .currentSection }}
+    {{- $contentType := .contentType }}
+    {{- $depthLimit := .depthLimit }}
+    {{- $depthCnt := add .depthCnt 1 }}
+
+    {{- with .startSection }}
+        <li>
+            {{- with $currentSection }}
+                {{- if (or (not (in $contentType "section")) .IsSection) }}
```

Comment:
Can you use `(eq $contentType "section")` here? The `in` invocation seems wasteful.

Reply:
That is indeed possible. Thanks for the tip.